### ISSUE TITLE
Killing more dangerous targets as first

### ISSUE THREAD

**Issue by michalrozek90** [Marsbars/DungeonCrawler-Issues#24]
There are many situations that the party should focus on something inside instance to kill first like those alarming robots in Gnomeregan, healing totems spawned by some mobs (those two are very low health) but it can be a simple mob-healer.

It is possible to change targets to smth like "healing ward" totem - which is healing all mobs around ?

**Comment by Marsbars** [comment]
Yeah, this makes sense however I wonder if it'll work with totems as I'm not sure whether they're things you'd be considered in combat with. You don't want the bots to just start attacking something they aren't in combat with and then potentially pulling something else

**Comment by michalrozek90** [comment]
It is possible to force the tank to mark those mobs / totems etc with a let's say skull and the rest will know that they need to kill a skull ?

**Comment by Marsbars** [comment]
Yeah I thought about the same thing however that would only work in a 5man team and also still not solve the issue? Imagine you're in combat with a group of 3 mobs and then something from the target list is in range but not in combat with you, you don't want to focus it right? If we do it only with mobs you're in combat with that would work. But the problem might still be totems as I'm not sure if they're considered in combat

**Comment by michalrozek90** [comment]
Yeah this should only considers units that are in combat after pull. So maybe the best would be pull nearest mob > and then search for a dangerous tarets to snipe. I will check in dev tools if the totems are in combat next time we will run dcrawler.
